ghost/i18n/locales/mn/comments.json

```diff
@@ -1,70 +1,70 @@
-    "edited": "",
-    "Enter your name": "",
-    "Expertise": "",
-    "Founder @ Acme Inc": "",
-    "Full-time parent": "",
-    "Head of Marketing at Acme, Inc": "",
-    "Hidden for members": "",
-    "Hide": "",
-    "Hide comment": "",
-    "Jamie Larson": "",
-    "Join the discussion": "",
-    "Just now": "",
-    "Load more ({{amount}})": "",
-    "Local resident": "",
-    "Member discussion": "",
-    "Name": "",
-    "Neurosurgeon": "",
-    "Once deleted, this comment can’t be recovered.": "",
-    "One hour ago": "",
-    "One min ago": "",
-    "removed": "",
-    "Replied to": "",
-    "Reply": "",
-    "Reply to": "",
-    "Reply to comment": "",
-    "Report": "",
-    "Report comment": "",
-    "Report this comment?": "",
-    "Save": "",
-    "Sending": "",
-    "Sent": "",
-    "Show": "",
-    "Show {{amount}} more replies": "",
-    "Show 1 more reply": "",
-    "Show comment": "",
-    "Sign in": "",
-    "Sign up now": "",
-    "Sort by": "",
-    "Start the conversation": "",
-    "This comment has been hidden.": "",
-    "This comment has been removed.": "",
-    "Upgrade now": "",
-    "Yesterday": "",
-    "Your request will be sent to the owner of this site.": ""
-}
+    "{{amount}} characters left": "{{amount}} тэмдэгт үлдсэн",
+    "{{amount}} comments": "{{amount}} сэтгэгдэл",
+    "{{amount}} hrs ago": "{{amount}} цагийн өмнө",
+    "{{amount}} mins ago": "{{amount}} минутын өмнө",
+    "{{amount}} more": "{{amount}} илүү",
+    "1 comment": "1 сэтгэгдэл",
+    "Add comment": "Сэтгэгдэл нэмэх",
+    "Add context to your comment, share your name and expertise to foster a healthy discussion.": "Сэтгэгдэлдээ илүү мэдээлэл оруулан, нэр болон мэргэжлээ хуваалцаж, үр бүтээлтэй хэлэлцүүлэг өрнүүлнэ үү.",
+    "Add reply": "Хариу нэмэх",
+    "Already a member?": "Бүртгэлтэй юу?",
+    "Anonymous": "Нууц",
+    "Are you sure?": "Та итгэлтэй байна уу?",
+    "Become a member of {{publication}} to start commenting.": "Сэтгэгдэл бичихийн тулд {{publication}}-н гишүүн болно уу.",
+    "Become a paid member of {{publication}} to start commenting.": "Сэтгэгдэл бичихийн тулд {{publication}}-н төлбөртэй гишүүн болно уу.",
+    "Cancel": "Цуцлах",
+    "Comment": "Сэтгэгдэл",
+    "Complete your profile": "Профайлаа бүрэн бөглөнө үү",
+    "Delete": "Устгах",
+    "Deleted": "Устгагдсан",
+    "Deleted member": "Устгагдсан гишүүн",
+    "Deleting": "Устгаж байна",
+    "Discussion": "Хэлэлцүүлэг",
+    "Edit": "Засах",
+    "Edit this comment": "Энэхүү сэтгэгдлийг засах",
+    "edited": "засагдсан",
+    "Enter your name": "Нэрээ оруулна уу",
+    "Expertise": "Мэргэжил",
+    "Founder @ Acme Inc": "Acme Inc-н үүсгэн байгуулагч",
+    "Full-time parent": "Бүтэн цагийн эцэг/эх",
+    "Head of Marketing at Acme, Inc": "Acme, Inc-н маркетингийн удирдлага",
+    "Hidden for members": "Гишүүдээс нуугдсан",
+    "Hide": "Нуух",
+    "Hide comment": "Сэтгэгдлийг нуух",
+    "Jamie Larson": "Жэйми Ларсон",
```

Comment:
The intent is to have a recognizable name here.  (Gender neutral if possible.)  I think you've got a transliteration?  Please pick something that would be recognizable as a name.